R/postprocess.R

```diff
@@ -1,3 +1,132 @@
+#' Extract samples of the delay distribution parameters
+#'
+#' @param fit A model fit with `epidist::epidist`
+#' @param ... Additional arguments passed to `brms::prepare_predictions`
```

Comment:
I think the modern way to link is [brms::prepare_predictions()] and I think this will result in working pkgdown links whilst what we have currently won't. Not 100% though. What do you think? If this is the case I guess its a new issue to check and do everywhere.

Reply:
Yes I think this is the way to do it and it should be everywhere. Overall I've been a bit lazy with documentation writing and when we get to 0.1.0 I think there is an overall sweep that should be done to improve